conference.js

```diff
@@ -1045,6 +1062,10 @@ export default {
                 (typeof config.hosts.jirecon != "undefined"))?
                 "jirecon" : "colibri";
         }
+        // Initialize in audio only mode
```

Comment:
Ok, so the problem is when there is channelLastN defined in config.js. It will overwrite the change done by audio only logic.